[refresh2020q1][rabbitmq] - Elixir error while building rabbitmq

This error is occuring during the refresh process. I have posted the context of the error and linked the file at the bottom of the issue.

Error:

```
** (Mix) You're trying to run :rabbitmqctl on Elixir v1.10.0 but it has declared in its mix.exs file it supports only Elixir >= 1.6.6 and < 1.10.0
make[2]: *** [Makefile:93: escript/rabbitmqctl] Error 1
make[2]: Leaving directory '/hab/cache/src/rabbitmq-server-3.7.16/deps/rabbitmq_cli'
make[1]: *** [erlang.mk:4322: deps] Error 2
make[1]: Leaving directory '/hab/cache/src/rabbitmq-server-3.7.16/deps/rabbit'
make: *** [erlang.mk:4322: deps] Error 2
   rabbitmq: Build time: 0m15s
   rabbitmq: Exiting on error
```

Steps to reproduce:

```
cd /tmp/
git clone --depth 1 -b refresh/2020q1_008 --single-branch https://github.com/habitat-sh/core-plans.git /tmp/core-plans"
wget https://gist.githubusercontent.com/MindNumbing/255ca89882213bf0a4422f28fb902570/raw/b00ed3638b73d8a697ca411ea5154bd77118d456/hab_studio.env
source hab.env
hab studio enter
build core-plans/rabbitmq
```

Full Error Log:
[rabbitmq.txt](https://github.com/habitat-sh/core-plans/files/4458568/rabbitmq.txt)

--- a/.refresh/remaining_plans_failed.txt
+++ b/.refresh/remaining_plans_failed.txt
@@ -22,7 +22,6 @@ core-plans/cockroach
 core-plans/buildkite-cli
 core-plans/shield
 core-plans/serf
-core-plans/rabbitmq
 core-plans/psscriptanalyzer
 core-plans/percona-xtrabackup
 core-plans/shared-mime-info

--- a/.refresh/remaining_plans_passed.txt
+++ b/.refresh/remaining_plans_passed.txt
@@ -362,6 +362,7 @@ core-plans/shield-proxy
 core-plans/sensu
 core-plans/scaffolding-ruby
 core-plans/relx
+core-plans/rabbitmq
 core-plans/prometheus-cpp
 core-plans/pester
 core-plans/opam

--- a/rabbitmq/plan.sh
+++ b/rabbitmq/plan.sh
@@ -1,13 +1,13 @@
 pkg_name=rabbitmq
 pkg_distname="${pkg_name}-server"
 pkg_origin=core
-pkg_version=3.7.16
+pkg_version=3.8.3
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('MPL')
 pkg_description="Open source multi-protocol messaging broker"
 pkg_upstream_url="https://www.rabbitmq.com"
 pkg_source="https://github.com/rabbitmq/rabbitmq-server/releases/download/v${pkg_version}/rabbitmq-server-${pkg_version}.tar.xz"
-pkg_shasum=7f7a8a587b6beba3b86e1a2a9bf60da4bd19b0aacfec025559bbbbf13fd1418b
+pkg_shasum=aedc8458701a80167b35958c10090d468c3de11984510b49ec02e777c51b1aba
 pkg_dirname="${pkg_distname}-${pkg_version}"
 pkg_deps=(
   core/coreutils

